inflate_p.h

```diff
@@ -211,3 +212,59 @@ static inline void window_output_flush(PREFIX3(stream) *strm) {
     state->whave += out_bytes;
     state->whave = MIN(state->whave, state->wsize);
 }
+
+/* Behave like chunkcopy, but avoid writing beyond of legal output. */
+static inline uint8_t* chunkcopy_safe(uint8_t *out, uint8_t *from, unsigned len, uint8_t *safe) {
+    unsigned safelen = (unsigned)((safe - out) + 1);
+    len = MIN(len, safelen);
+    int olap_src = ((from >= out && from < out+len) ||
+            (from + len > out && from <= out));
```

Comment:
Can you fix indentation of these two to match the bottom two?

Reply:
Also add space between `out+len`

Reply:
Heh, caught it in the latest push.